crates/stc_ts_builtin_macro/src/lib.rs

```diff
@@ -5,20 +5,20 @@
 #[macro_use]
 extern crate pmutil;
 
-use std::{collections::HashMap, fs::read_dir, path::Path, sync::Arc};
+use std::{collections::HashMap, env, fs::read_dir, path::Path};
 
 use inflector::Inflector;
 use pmutil::Quote;
 use proc_macro2::Span;
-use swc_common::{comments::SingleThreadedComments, FilePathMapping, SourceMap};
+use swc_common::{comments::SingleThreadedComments, sync::Lrc, FilePathMapping, SourceMap};
 use swc_ecma_parser::{lexer::Lexer, Parser, StringInput, Syntax, TsConfig};
 use swc_macros_common::{call_site, print};
 use syn::{punctuated::Punctuated, Token};
 
 #[proc_macro]
 pub fn builtin(_: proc_macro::TokenStream) -> proc_macro::TokenStream {
     swc_common::GLOBALS.set(&swc_common::Globals::new(), || {
-        let cm = Arc::new(SourceMap::new(FilePathMapping::empty()));
+        let cm = Lrc::new(SourceMap::new(FilePathMapping::empty()));
```

Comment:
It was complaining about `Arc` on a non-Send/Sync type, interestingly. I think that's probably a clippy bug? Strictly, you don't even need an rc at all, but the docs for SourceMap suggest using this alias, which suppresses the lint.